src/types/gameConsole.ts

```diff
@@ -12,109 +12,116 @@ export default class GameConsole {
+    new GameConsole(/7800/, ['.a78'], '7800', 'Atari7800', 'SEVENTYEIGHTHUNDRED', 'atari7800'),
+    new GameConsole(/Jaguar/i, ['.j64'], undefined, undefined, 'JAGUAR', 'jaguar'),
+    new GameConsole(/Lynx/i, ['.lnx', '.lyx'], undefined, 'AtariLynx', 'LYNX', 'lynx'),
+    new GameConsole(/Atari (- )?ST/i, ['.msa', '.st', '.stx'], undefined, 'AtariST', 'ATARIST', 'atarist'),
     // Bally
-    new GameConsole(/Astrocade/i, [/* '.bin' */], undefined, 'Astrocade', undefined),
+    new GameConsole(/Astrocade/i, [/* '.bin' */], undefined, 'Astrocade', undefined, 'astrocde'),
     // Bandai
-    new GameConsole(/WonderSwan/i, ['.ws'], 'wonderswan', 'WonderSwan', 'WS'),
-    new GameConsole(/WonderSwan Color/i, ['.wsc'], 'wonderswan', 'WonderSwan', 'WS'),
+    new GameConsole(/WonderSwan/i, ['.ws'], 'wonderswan', 'WonderSwan', 'WS', 'wswan'),
+    new GameConsole(/WonderSwan Color/i, ['.wsc'], 'wonderswan', 'WonderSwan', 'WS', 'wswanc'),
     // Bit Corporation
-    new GameConsole(/Gamate/i, [/* '.bin' */], 'gamate', 'Gamate', undefined),
+    new GameConsole(/Gamate/i, [/* '.bin' */], 'gamate', 'Gamate', undefined, 'gamate'),
     // Capcom
     // TODO(cemmer): CPS1, CPS2, CPS3
     // Casio
-    new GameConsole(/PV-?1000/i, [/* '.bin' */], undefined, 'Casio_PV-1000', undefined),
+    new GameConsole(/PV-?1000/i, [/* '.bin' */], undefined, 'Casio_PV-1000', undefined, 'pv1000'),
     // Commodore
-    new GameConsole(/Amiga/i, [], 'amiga', 'Amiga', 'AMIGA'),
-    new GameConsole(/Commodore 64/i, ['.crt', '.d64', '.t64'], undefined, 'C64', 'COMMODORE'),
+    // TODO(dcramer): multiple outputs for Batocera
+    new GameConsole(/Amiga/i, [], 'amiga', 'Amiga', 'AMIGA', undefined),
+    new GameConsole(/Commodore 64/i, ['.crt', '.d64', '.t64'], undefined, 'C64', 'COMMODORE', 'c64'),
     // Coleco
-    new GameConsole(/ColecoVision/i, ['.col'], 'coleco', 'Coleco', 'COLECO'),
+    new GameConsole(/ColecoVision/i, ['.col'], 'coleco', 'Coleco', 'COLECO', 'colecovision'),
     // Emerson
-    new GameConsole(/Arcadia/i, [/* '.bin' */], 'arcadia', 'Arcadia', undefined),
+    new GameConsole(/Arcadia/i, [/* '.bin' */], 'arcadia', 'Arcadia', undefined, 'arcadia'),
     // Entex
-    new GameConsole(/Adventure Vision/i, [/* '.bin' */], 'avision', 'AVision', undefined),
+    new GameConsole(/Adventure Vision/i, [/* '.bin' */], 'avision', 'AVision', undefined, 'advision'),
     // Fairchild
-    new GameConsole(/Channel F/i, [/* '.bin' */], 'channel_f', 'ChannelF', 'FAIRCHILD'),
+    new GameConsole(/Channel F/i, [/* '.bin' */], 'channel_f', 'ChannelF', 'FAIRCHILD', 'channelf'),
     // GCE
-    new GameConsole(/Vectrex/i, ['.vec'], undefined, 'Vectrex', 'VECTREX'),
+    new GameConsole(/Vectrex/i, ['.vec'], undefined, 'Vectrex', 'VECTREX', 'vectrex'),
     // Interton
-    new GameConsole(/VC ?4000/i, [/* '.bin' */], undefined, 'VC4000', undefined),
+    new GameConsole(/VC ?4000/i, [/* '.bin' */], undefined, 'VC4000', undefined, 'vc4000'),
```

Comment:
I'm not seeing this one in https://wiki.batocera.org/systems, but it looks like you found this in your install?

Reply:
Maybe here: https://github.com/batocera-linux/batocera.linux/blob/master/batocera-Changelog.md#20220524---batoceralinux-34---blue-morpho